cli50/__main__.py

```diff
@@ -163,6 +163,10 @@ def main():
     # Check Docker Hub for newer image
     if not args["fast"]:
 
+        # Determine platform architecture
+        arch = subprocess.check_output(["uname", "-m"], stderr=subprocess.DEVNULL).decode("utf-8").strip()
+        arch = "amd64" if arch == "x86_64" else "arm64"
```

Comment:
This might not work on Windows? (Though untested as is.) Is there a way to ask Docker to determine the current architecture instead, to avoid hard-coding these two, in case we add others in the future?

Reply:
Actually, no need to detect platform architecture. I think we can just iterate through the image digests in the manifest and see if there is a match with the remote image digest. If there is none, prompt the user to pull a new image.

Reply:
This is because when docker fetch the remote image, it will automatically get the compatible image.

Reply:
OK, a second thought, determining platform architecture might be cleaner.

Reply:
> This might not work on Windows? (Though untested as is.) Is there a way to ask Docker to determine the current architecture instead, to avoid hard-coding these two, in case we add others in the future?

Indeed `uname` was not supported on windows. Will use docker command to get the platform architecture.